native_client/test/concurrent_streams.py

```diff
@@ -53,13 +53,15 @@ def main():
     if args.lm and args.trie:
         ds.enableDecoderWithLM(args.alphabet, args.lm, args.trie, LM_ALPHA, LM_BETA)
 
-    with wave.open(args.audio1, 'rb') as fin:
-        fs1 = fin.getframerate()
-        audio1 = np.frombuffer(fin.readframes(fin.getnframes()), np.int16)
-
-    with wave.open(args.audio2, 'rb') as fin:
-        fs2 = fin.getframerate()
-        audio2 = np.frombuffer(fin.readframes(fin.getnframes()), np.int16)
+    fin = wave.open(args.audio1, 'rb')
```

Comment:
Why not using `with` ?

Reply:
Python 2.7 wave.open objects don't support that syntax.